bedrock/thunderbird/helpers.py

```diff
@@ -1,10 +1,101 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
 import jingo
+import jinja2
 
+from bedrock.thunderbird.details import thunderbird_desktop
 from bedrock.base.urlresolvers import reverse
+from lib.l10n_utils import get_locale
+
+
+@jingo.register.function
+@jinja2.contextfunction
+def download_thunderbird(ctx, channel='release', small=False, icon=True,
+                         dom_id=None, locale=None, simple=False,
+                         force_direct=False):
+    """ Output a "Download Thunderbird" button.
+
+    :param ctx: context from calling template.
+    :param channel: name of channel: 'release', 'beta' or 'alpha'.
+    :param small: Display the small button if True.
+    :param icon: Display the Fx icon on the button if True.
+    :param dom_id: Use this string as the id attr on the element.
+    :param locale: The locale of the download. Default to locale of request.
+    :param simple: Display button with text only if True. Will not display
+            icon or privacy/what's new/systems & languages links. Can be used
+            in conjunction with 'small'.
```

Comment:
Looks like the systems & languages/privacy/what's new links show even with `simple=true`, so maybe update this comment?